relays/substrate/src/main.rs

```diff
@@ -569,7 +724,30 @@ impl crate::cli::ToMillauMessage {
 			cli::ToMillauMessage::Transfer { recipient, amount } => {
 				millau_runtime::Call::Balances(millau_runtime::BalancesCall::transfer(recipient, amount))
 			}
-		}
+		};
+
+		log::info!(target: "bridge", "Generated Millau call: {:#?}", call);
+		log::info!(target: "bridge", "Weight of Millau call: {}", call.get_dispatch_info().weight);
+		log::info!(target: "bridge", "Encoded Millau call: {:?}", HexBytes::encode(&call));
+
+		Ok(call)
+	}
+}
+
+/// Nicer formatting for raw bytes vectors.
+#[derive(Encode, Decode)]
+struct HexBytes(Vec<u8>);
+
+impl Debug for HexBytes {
+	fn fmt(&self, fmt: &mut std::fmt::Formatter) -> std::fmt::Result {
+		write!(fmt, "0x{}", hex::encode(&self.0))
```

Comment:
```suggestion
		write!(fmt, "{:#x?}", hex::encode(&self.0))
```

Reply:
Encode returns a `String`, why would I hex-encode that?

Reply:
I was trying to save you the trouble of manually prepending the `0x` - but yeah, didn't realize the formatter wouldn't work on `String`s